pkg/util/steps/refreshing.go

```diff
@@ -84,6 +86,19 @@ func (s *authorizationRefreshingActionStep) run(ctx context.Context, log *logrus
 		return true, err
 	}, timeoutCtx.Done())
 
+	// After timeout, return any actionable errors to the user
+	switch err != nil {
```

Comment:
switch has to be on a var.  we're getting a nil pointer exception on this.   I'd just suggest to use if else statements.